proposals/ref-readonly-parameters.md

```diff
@@ -115,6 +115,8 @@ This was discussed in [LDM 2022-05-11](https://github.com/dotnet/csharplang/blob
 ### Pending LDM review
 [to-review]: #pending-ldm-review
 
+Inverse ordering of modifiers (`readonly ref` instead of `ref readonly`) could be allowed.
```

Comment:
May want to add a note that this could cause issues if we wanted readonly parameters, and would be inconsistent with ref fields and ref returns.

Reply:
Yeah I'd also personally prefer if the order here was strict. It'd also match all other uses of `readonly`, where the exact position matters when applied to a `ref`. Eg. you can't have a `readonly readonly ref` method. Or a `readonly readonly ref` field. Similarly, a `readonly ref` return or a `readonly ref` field have a very different meaning than a `ref readonly` return or field. It would seem weird if the same didn't apply to parameters as well 🤔